PocketKit/Tests/PocketKitTests/Support/MockPocketBraze.swift

```diff
@@ -46,4 +46,8 @@ extension MockPocketBraze {
 
         return calls[index] as? DidRecieveUserNotifcationImpl
     }
+
+    func signedInUserDidBeginMigration() {
+
+    }
```

Comment:
is this needed

Reply:
It's part of the protocol so it must be implemented in the mock.